src/base_object-inl.h

```diff
@@ -123,6 +123,22 @@ BaseObject::MakeLazilyInitializedJSTemplate(Environment* env) {
   return t;
 }
 
+template <int kField>
```

Comment:
This is not a constant, so `Field` is enough?

Reply:
@joyeecheung I thought this was our common style for our codebase, but apparently it’s not. Done!